extras/AFC_hub.py

```diff
@@ -47,10 +47,76 @@ def handle_connect(self):
         self.gcode = self.AFC.gcode
         self.reactor = self.AFC.reactor
 
+        #self.gcode.register_command('HUB_CUT_TEST', self.cmd_HUB_CUT_TEST, desc=self.cmd_HUB_CUT_TEST_help)
```

Comment:
Uncomment both lines?

Reply:
So thinking about this more, wouldn't it make sense to keep the register commands in AFC.py and a cmd_ function in AFC.py that then looks up current lane and calls CUR_LANE.unit.hub_cut_test / hub_load functions. This would be the same way that TOOL_LOAD/UNLOAD would work